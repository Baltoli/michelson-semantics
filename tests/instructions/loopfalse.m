{ code { DROP ; 
         PUSH bool False ;
         LOOP { FAILWITH False } ;
         PUSH bool False ;
         NIL operation ;
         PAIR }; 
storage unit;
parameter unit; } Unit Unit
