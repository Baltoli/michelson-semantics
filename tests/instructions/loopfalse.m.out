<generatedTop>
  <k>
    .
  </k>
  <stack>
    Stack ( ListItem ( Pair { .DataList } false ) )
  </stack>
</generatedTop>
